### DISCUSSION TITLE
In Sub Flows inside the Edge Connections is not selecting or Clickable.

### DISCUSSION THREAD

**Discussion by sudarsanarao995** [xyflow/xyflow#3498] [General]
I Followed this link for Sub Flows
https://reactflow.dev/docs/guides/sub-flows/

here the sandbox code

https://codesandbox.io/s/43xvxd?file=/nodes.js&utm_medium=sandpack

Here inside the subflow node connections , I unable to not clickable or selectable . Please observe the below image

![image](https://github.com/wbkd/react-flow/assets/147405464/0c4c42a8-a74d-4b18-9a14-b56ffc0b82af)

**Excepting Result that Inside the subflow should be Clickable and select**

**Comment by moklick** [comment]
by default all edges get rendered below the nodes. If you want to change that, you can use the zIndex edge option like this: https://codesandbox.io/s/late-frost-hl5hhp?file=/edges.js:18-309

**Reply by sudarsanarao995** [reply]
Thank you working as excepted
